.gitlab/benchmarks/scripts/analyze-results.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
We already have a `test/benchmarks/` directory which is used for some benchmark-related files. Could you:
* Create a subdirectory there for your files
* Add yourself as CODEOWNERS of this subdirectory
* Move all the scripts there
We'd like to keep the `.gitlab/` directory for Gitlab config files only

Reply:
Great idea, I've created a subdir called `apm_scripts` there and assigned us code ownership